scripts/quota_log.py

```diff
@@ -71,25 +90,54 @@ def save_results(disk_quota_data, cpu_hours_data, db_config):
             merged_results[project] = cpu_hours_data[project]
 
 
+    logging.info("Connecting to the Database: " + db_config['url'])
     # create db instance
     server = "http://{username}:{password}@{url}:{port}".format(
         url=db_config['url'],
         username=db_config['username'],
         password=db_config['password'],
         port=db_config['port'])
+    try:
+        couch = couchdb.Server(server)
+    except Exception, e:
+        logging.error(e.message)
+        exit()
 
-    couch = couchdb.Server(server)
     db = couch['uppmax']
+    logging.info('Connection established')
 
     # save results in the database
     for project in merged_results.values():
-        db.save(project)
+        try:
+            project_id, project_rev = db.save(project)
+        except Exception, e:
+            logging.error(e.message)
+            exit()
+        else:
+            logging.info('Project:' +  project_id, project['project'] + ' has been updated')
```

Comment:
same thing here, try to use .format over concat with plusses. That's why the langage maintainer asks for, and it's slighly more efficient.